resources/translations/fi.edn

```diff
@@ -278,6 +278,9 @@
                 :column-key "Tunniste"
                 :column-label "Otsikko"
                 :confirm-remove-field "Poista kenttä?"
+                :date-bound {:none "Ei rajausta"
+                              :past "Menneet"
+                              :future "Tulevat"}
```

Comment:
Same indentation problem here. Perhaps `cljfmt` is not run to `edn` files at the moment.